internal/util/volumes_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 - 2025 Crunchy Data Solutions, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"gotest.tools/v3/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/crunchydata/postgres-operator/internal/testing/cmp"
+)
+
+func TestAddVolumeAndMountsToPod(t *testing.T) {
+	pod := &corev1.PodSpec{
+		Containers: []corev1.Container{
```

Comment:
🔧 Might be worth throwing in 2+ initContainers to show that this func adds to those containers as well

Reply:
Good call.